packages/api/src/platforms/vtex/resolvers/seo.ts

```diff
@@ -1,6 +1,10 @@
 import type { Resolver } from '..'
+import type { PromiseType } from '../../../typings'
+import type { StoreProduct } from './product'
 
-export type Root = { title?: string; description?: string; canonical?: string }
+export type Root = PromiseType<ReturnType<typeof StoreProduct.seo>> & {
```

Comment:
just to use the right dynamic type instead of the fixed one. Not related to this PR.